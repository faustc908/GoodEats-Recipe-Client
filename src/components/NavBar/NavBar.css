.navbar {
  padding: 5px;
  font-size: 25px;
  background-color: rgb(255, 255, 255);
  font-family: sans-serif;
  height: 10vh;
  color: rgb(177, 245, 245);
}

.icon {
  padding-right: 10px;
  color: gray;
}

.texta {
  float: left;
  color: rgba(0, 0, 0, 0.601);
  font-style: italic;
}

.textb {
  text-align: right;
  float: right;
  font-style: italic;
  color: rgba(49, 45, 45, 0.601);
}

.favicon {
  height: 15px;
  width: 15px;
  float: left;
}
