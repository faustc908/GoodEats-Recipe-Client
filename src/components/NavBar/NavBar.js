import React from "react";
import "./NavBar.css";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";

// NavBar with favicon

class NavBar extends React.Component {
  render() {
    return (
      <div className="navbar">
        <p className="texta">
          <FontAwesomeIcon className="icon" icon="utensils" />
          GoodEats
        </p>
        <p className="textb">Find new recipes</p>
      </div>
    );
  }
}

export default NavBar;
