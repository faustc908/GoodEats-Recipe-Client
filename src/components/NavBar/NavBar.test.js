import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NavBar from './NavBar'
import ReactDOM from 'react-dom';

describe(`NavBar component`, () => {

    it('renders a NavBar by default', () => {
        const wrapper = shallow(<NavBar />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<NavBar />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
})