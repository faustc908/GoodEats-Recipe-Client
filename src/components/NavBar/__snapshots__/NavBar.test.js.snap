// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component renders a NavBar by default 1`] = `
<div
  className="navbar"
>
  <p
    className="texta"
  >
    <FontAwesomeIcon
      border={false}
      className="icon"
      fixedWidth={false}
      flip={null}
      icon="utensils"
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
    GoodEats
  </p>
  <p
    className="textb"
  >
    Find new recipes
  </p>
</div>
`;
